lib/ipfs.js

```diff
@@ -0,0 +1,11 @@
+// TEMPORARY FILE UNTIL THESE METHODS ARE ADDED TO NPM PACKAGE.
```

Comment:
does this have to be a `js` file? everything else is `ts`

Reply:
updated to `.ts` ✅